client/my-sites/domains/domain-management/dns/dns-records-list.jsx

```diff
@@ -230,12 +231,13 @@ class DnsRecordsList extends Component {
 	}
 
 	render() {
-		const { dns, selectedDomainName, selectedSite } = this.props;
+		const { dns, selectedDomain, selectedDomainName, selectedSite } = this.props;
 		const { dialog } = this.state;
 
 		let domainConnectRecordIsEnabled = false;
 		const dnsRecordsList = dns.records.map( ( dnsRecord, index ) => {
-			if ( 'NS' === dnsRecord.type ) {
+			// We want to hide root NS records for root domains, but not for subdomains
+			if ( 'NS' === dnsRecord.type && ! selectedDomain.isSubdomain && isRootRecord ) {
```

Comment:
`isRootRecord` is undefined here. 🤔

Reply:
Oops 😬 forgot to add this line when committing